src/global/testing/jest-setup.ts

```diff
@@ -1,5 +1,7 @@
 import { mergeConfig, SbbIconConfig } from '../config';
 
+// TODO-Migr Find a way to reuse this setup for tests
```

Comment:
It might be possible to use the `testRunnerHtml` callback of the web-test-runner group configuration, to apply this configuration.

Reply:
Yep, that's the idea. I'll do that asap